packages/block-library/src/file/inspector.js

```diff
@@ -2,16 +2,30 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { PanelBody, SelectControl, ToggleControl } from '@wordpress/components';
+import {
+	PanelBody,
+	RangeControl,
+	SelectControl,
+	ToggleControl,
+} from '@wordpress/components';
 import { InspectorControls } from '@wordpress/block-editor';
 
+/**
+ * Internal dependencies
+ */
+import { MIN_PREVIEW_HEIGHT, MAX_PREVIEW_HEIGHT } from './edit';
+
 export default function FileBlockInspector( {
 	hrefs,
 	openInNewWindow,
 	showDownloadButton,
 	changeLinkDestinationOption,
 	changeOpenInNewWindow,
 	changeShowDownloadButton,
+	displayPreview,
+	changeDisplayPreview,
+	previewHeight,
```

Comment:
To avoid errors for existing files.
```suggestion
	previewHeight = MIN_PREVIEW_HEIGHT,
```

Reply:
Resolved by setting a default height, instead.